fix: skia mobile StatusBar styling

rajamatt opened the pull request:
**GitHub Issue:** closes https://github.com/unoplatform/uno-private/issues/1251

<!-- Link to relevant GitHub issue if applicable. All PRs should be associated with an issue (GitHub issue or internal), unless the change is documentation related. -->

## PR Type:

- 🐞 Bugfix

## What is the current behavior? 🤔

<!-- Please describe the current behavior that you are modifying, or link to a relevant issue. -->

Android:

1) No support for edge-to-edge even though its [enabled by default in Android 15+ and when using net9.0](https://github.com/unoplatform/uno/blob/2c73a4dde0b0b0cca22da964081ea053f0770d47/src/Uno.UI/FeatureConfiguration.cs#L994-L1022)

iOS:

1) StatusBar theme changing not working because needs the [`UIViewControllerBasedStatusBarAppearance`](https://developer.apple.com/documentation/bundleresources/information-property-list/uiviewcontrollerbasedstatusbarappearance) to be false in the Info.plist

## What is the new behavior? 🚀

<!-- Please describe the new behavior after your modifications. -->

Android:

1) Support for edge-to-edge StatusBar for Android 15+ and when using net9.0
2) Now we use WindowInsetsCompat to control the BackgroundColor of the StatusBar

iOS:

1) Set `UIViewControllerBasedStatusBarAppearance` to false in the samples so that StatusBar theme changing works (we already set it to false in new apps with the recommended template)
2) Now we add a subView on the StatusBar to change its BackgroundColor, similarly to whats happening with edge-to-edge for Android

## PR Checklist ✅

Please check if your PR fulfills the following requirements:

- [x] 📝 Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.
- [x] 🧪 Added [Runtime tests, UI tests, or a manual test sample](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [x] 📚 Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] 🖼️ Validated PR `Screenshots Compare Test Run` results.
- [x] ❗ Contains **NO** breaking changes

<!-- If this PR contains a breaking change, please describe the impact and migration path for existing applications below.
     Please note that breaking changes are likely to be rejected -->

## Other information ℹ️

<!-- Please provide any additional information if necessary -->

Changing the BackgroundColor & ForegroundColor of the StatusBar should be working with both native and Skia renderers. The colors should update automatically, without needing to force a refresh of the visual states. There are differences in the code between Android >= 15 and Android < 15.

kazo0 committed:
fix: skia mobile StatusBar styling

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-20687/docs/index.html

unodevops commented:
🤖 Your WebAssembly Skia Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-20687/wasm-skia-net9/index.html

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-20687/docs/index.html

rajamatt committed:
chore: add statusbarcolor tests page

rajamatt committed:
chore: revert titlebarcolortests page

unodevops commented:
🤖 Your WebAssembly Skia Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-20687/wasm-skia-net9/index.html

unodevops commented:
🤖 Your WebAssembly Skia Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-20687/wasm-skia-net9/index.html

nventive-devops commented:
The [build 167617](https://dev.azure.com/uno-platform/Uno%20Platform/_build/results?buildId=167617) found UI Test snapshots differences: `android-28-net8`: **25**, `android-28-net8-Snap`: **42**, `ios`: **255**, `ios-Snap`: **984**, `skia-linux-screenshots`: **78**, `skia-windows-screenshots`: **251**, `wasm`: **46**, `wasm-automated-net9.0-WinUI-Benchmarks-automated`: **1**, `wasm-automated-net9.0-WinUI-Default-automated`: **13**, `wasm-automated-net9.0-WinUI-RuntimeTests-0`: **0**, `wasm-automated-net9.0-WinUI-RuntimeTests-1`: **1**, `wasm-automated-net9.0-WinUI-RuntimeTests-2`: **1**

<details>
<summary>Details</summary>


* `android-28-net8`: **25** changed over 824

  <details>
  <summary>🚨🚨 Comparison Details (first 20) 🚨🚨</summary>

  - `ContentPresenter_Changing_ContentTemplate_Uno_UI_Samples_Content_UITests_ContentPresenter_ContentPresenter_Changing_ContentTemplate`
  - `Detereminate_ProgressRing_Validation75_[#FF0000_#008000_#008000_#008000]_Progress-Ring-Value-75`
  - `FocusManager_GetFocusedElement_Border_Validation_FocusManager_-_GetFocusedElement_-_Border_-_2_-_After_Selection`
  - `FocusManager_GetFocusedElement_Border_Validation_Uno_UI_Samples_Content_UITests_FocusTests_FocusManager_GetFocus_Automated`
  - `Detereminate_ProgressRing_Validation25_[#FF0000_#008000_#FF0000_#FF0000]_Progress-Ring-Value-25`
  - `FocusManager_GetFocusedElement_Border_Validation_FocusManager_-_GetFocusedElement_-_Border_-_1_-_Initial_State`
  - `ListView_SelectedItems_SamplesApp_Windows_UI_Xaml_Controls_ListView_ListViewSelectedItems`
  - `ProgressRing_IsEnabled_Running_UITests_Windows_UI_Xaml_Controls_ProgressRing_WindowsProgressRing_GH1220`
  - `ProgressRing_Visibility_Collapsed_UITests_Windows_UI_Xaml_Controls_ProgressRing_WindowsProgressRing_GH1220`
  - `When_Reading_Is_Attached_And_Detaches_UITests_Shared_Windows_Devices_GyrometerTests`
  - `UpDownEnabledTest_UITests_Shared_Microsoft_UI_Xaml_Controls_NumberBoxTests_NumberBoxPage`
  - `When_Reading_Is_Attached_And_Waits_UITests_Shared_Windows_Devices_GyrometerTests`
  - `WebView_NavigateToAnchor_Initial`
  - `When_Parent_PointerMoved_After_drag_on_non-scrolling_ScrollViewer`
  - `When_SingleSelectionWithItemClick_Then_PointersEvents_UITests_Windows_UI_Xaml_Controls_ListView_ListView_Selection_Pointers`
  - `When_ExtendedSelectionWithoutItemClick_Then_PointersEvents_UITests_Windows_UI_Xaml_Controls_ListView_ListView_Selection_Pointers`
  - `When_SingleSelectionWithoutItemClick_Then_PointersEvents_UITests_Windows_UI_Xaml_Controls_ListView_ListView_Selection_Pointers`
  - `When_MultipleSelectionWithoutItemClick_Then_PointersEvents_UITests_Windows_UI_Xaml_Controls_ListView_ListView_Selection_Pointers`
  - `Detereminate_ProgressRing_Validation50_[#FF0000_#008000_#008000_#FF0000]_Progress-Ring-Value-50`
  - `NativeCommandBar_Size_Uno_UI_Samples_Content_UITests_CommandBar_CommandBar_Dynamic`
  </details>

* `android-28-net8-Snap`: **42** changed over 1069

  <details>
  <summary>🚨🚨 Comparison Details (first 20) 🚨🚨</summary>

  - `MUX_NumberBox_UITests_Shared_Microsoft_UI_Xaml_Controls_NumberBoxTests_NumberBoxPage`
  - `Brushes_ImageBrushChangingCornerRadius_Uno_UI_Samples_UITests_ImageBrushTestControl_ImageBrushChangingCornerRadius`
  - `CommandBar_Extensions_Uno_UI_Samples_Content_UITests_CommandBar_CommandBar_Extensions`
  - `Gesture_Recognizer_Pointer_Events_test_bench_UITests_Shared_Windows_UI_Input_GestureRecognizer_PointersEvents`
  - `ListView_ListViewSelectedItems_SamplesApp_Windows_UI_Xaml_Controls_ListView_ListViewSelectedItems`
  - `TeachingTip_UITests_Microsoft_UI_Xaml_Controls_TeachingTipTests_TeachingTipPage_UITests_Microsoft_UI_Xaml_Controls_TeachingTipTests_TeachingTipPage`
  - `Border_Border_Simple_Uno_UI_Samples_UITests_BorderTestsControl_Border_Simple`
  - `CommandBar_CommandBar_BackButtonTitle_UITests_Windows_UI_Xaml_Controls_CommandBar_BackButtonTitle_CommandBar_Frame`
  - `CommandBar_CommandBar_Xaml_Automated_UITests_Shared_Windows_UI_Xaml_Controls_CommandBar_CommandBar_Xaml_Automated`
  - `CommandBar_Examples_Uno_UI_Samples_Content_UITests_CommandBar_CommandBar_Examples`
  - `ListView_ListView_With_ListViews_Count_Measure_UITests_Shared_Windows_UI_Xaml_Controls_ListView_ListView_With_ListViews_Count_Measure`
  - `NavigationView_MUXControlsTestApp_NavigationViewRS4Page_MUXControlsTestApp_NavigationViewRS4Page`
  - `ComboBox_UITests_Windows_UI_Xaml_Controls_ComboBox_ComboBox_Description_UITests_Windows_UI_Xaml_Controls_ComboBox_ComboBox_Description`
  - `CommandBar_BackGesture_Uno_UI_Samples_Content_UITests_CommandBar_CommandBar_BackGesture`
  - `Border_Border_CornerRadius_Gradient_UITests_Windows_UI_Xaml_Controls_BorderTests_Border_CornerRadius_Gradient`
  - `Icons_UITests_Windows_UI_Xaml_Controls_SymbolIconTests_SymbolIcon_Generic_UITests_Windows_UI_Xaml_Controls_SymbolIconTests_SymbolIcon_Generic`
  - `Transform_Basics_UITests_Shared_Windows_UI_Xaml_Media_Transform_Basics`
  - `Brushes_GenericApp_Views_Content_UITests_ImageBrushTestControl_ImageBrush_UniformToFill_GenericApp_Views_Content_UITests_ImageBrushTestControl_ImageBrush_UniformToFill`
  - `ComboBox_UITests_Shared_Windows_UI_Xaml_Controls_ComboBox_ComboBox_VisibleBounds_UITests_Shared_Windows_UI_Xaml_Controls_ComboBox_ComboBox_VisibleBounds`
  - `MediaPlayerElement_Mini_player_UITests_Shared_Windows_UI_Xaml_Controls_MediaPlayerElement_MediaPlayerElement_Minimal`
  </details>

* `ios`: **255** changed over 255

  <details>
  <summary>🚨🚨 Comparison Details (first 20) 🚨🚨</summary>

  - `Button_NestedButtons_Validation_nVentive_Umbrella_Views_UI_Samples_Content_UITests_ButtonTestsControl_Nested_Buttons`
  - `BasicExpressionTest_UITests_Microsoft_UI_Xaml_Controls_NumberBoxTests_NumberBox_ExpressionTest`
  - `Check_ListView_Swallows_Measure_UITests_Shared_Windows_UI_Xaml_Controls_ListView_ListView_With_ListViews_Count_Measure`
  - `Check_ListView_Swallows_Measure_before_scroll`
  - `FlyoutTest_When_NoOverlayInputPassThroughElement_Then_DontPassThrough_woAuto_UITests_Shared_Windows_UI_Xaml_Controls_Flyout_Flyout_OverlayInputPassThroughElement`
  - `DecimalFormatterTest_UITests_Shared_Microsoft_UI_Xaml_Controls_NumberBoxTests_NumberBoxPage`
  - `Image_Source_Nullify_UITests_Windows_UI_Xaml_Controls_ImageTests_Image_Source_Nullify`
  - `Keyboard_Textbox_NoScrollViewer_Validation_1_-_Remove_Focus_on_filledTextBox`
  - `Keyboard_Textbox_NoScrollViewer_Validation_Uno_UI_Samples_Content_UITests_TextBoxControl_Input_Test_NoScrollViewer_Automated`
  - `ListView_ExpandableItemLarge_ExpandHeaderWithSingleItem_Validation_Initial_State`
  - `ListView_ExpandableItemLarge_ExpandHeader_Validation_SamplesApp_Windows_UI_Xaml_Controls_ListView_ListView_Expandable_Item_Large`
  - `ListView_ListViewVariableItemHeightLong_InitializesTest_SamplesApp_Windows_UI_Xaml_Controls_ListView_ListViewVariableItemHeightLong`
  - `ListView_ObservableCollection_Creation_Count_Scrolled1`
  - `ListView_VirtualizePanelAdaptaterIdCache_SamplesApp_Windows_UI_Xaml_Controls_ListView_ListView_VirtualizePanelAdaptaterIdCache`
  - `Pivot_Non_PivotItemChange_Validation_UITests_Shared_Windows_UI_Xaml_Controls_Pivot_Pivot_CustomContent_Automated`
  - `Screenshots_Image_Stretch_Alignment_Bigger_Mode-0`
  - `Screenshots_Image_Stretch_Alignment_Bigger_Mode-2`
  - `TextBox_AutoGrow_Vertically_Wrapping_Test_Uno_UI_Samples_Content_UITests_TextBoxControl_Input_Multiline_AutoHeight`
  - `Screenshots_Image_Stretch_Alignment_Bigger_Mode-3`
  - `TextBox_BeforeTextChanging_UITests_Shared_Windows_UI_Xaml_Controls_TextBoxTests_TextBox_BeforeTextChanging`
  </details>

* `ios-Snap`: **984** changed over 986

  <details>
  <summary>🚨🚨 Comparison Details (first 20) 🚨🚨</summary>

  - `Border_Border_AntiAlias_UITests_Windows_UI_Xaml_Controls_BorderTests_BorderAntiAlias`
  - `Border_Border_Simple_No_Background_With_TextBox_Uno_UI_Samples_UITests_BorderTestsControl_Border_Simple_No_Background_With_TextBox`
  - `Border_OnepxThicknessandRadiusAndTextBlock_Uno_UI_Samples_UITests_BorderTestsControl_OnepxThicknessandRadiusAndTextBlock`
  - `Border_Uniform_Thickness_and_Large_Radius_Uno_UI_Samples_UITests_BorderTestsControl_Uniform_Thickness_and_Large_Radius`
  - `Brushes_ImageBrushShapeStretchesAlignments_UITests_Windows_UI_Xaml_Media_ImageBrushTests_ImageBrushShapeStretchesAlignments`
  - `Brushes_PathImageBrushfill_Uno_UI_Samples_UITests_ImageBrushTestControl_PathImageBrushfill`
  - `Brushes_UITests_Windows_UI_Xaml_Media_GradientBrushTests_LinearGradientBrush_Change_Stops_UITests_Windows_UI_Xaml_Media_GradientBrushTests_LinearGradientBrush_Change_Stops`
  - `Brushes_Uno_UI_Samples_UITests_ImageBrushTestControl_ImageBrushAlignmentXY2_Uno_UI_Samples_UITests_ImageBrushTestControl_ImageBrushAlignmentXY2`
  - `Buttons_Hyperlink_Disabled_nVentive_Umbrella_Views_UI_Samples_Content_UITests_ButtonTestsControl_Hyperlink_Disabled`
  - `Buttons_Nested_Buttons_nVentive_Umbrella_Views_UI_Samples_Content_UITests_ButtonTestsControl_Nested_Buttons`
  - `Clipping_UITests_Shared_Windows_UI_Xaml_Clipping_ClippingRoundedCorners_UITests_Shared_Windows_UI_Xaml_Clipping_ClippingRoundedCorners`
  - `Clipping_UITests_Shared_Windows_UI_Xaml_Clipping_ClippingToControlBounds_UITests_Shared_Windows_UI_Xaml_Clipping_ClippingToControlBounds`
  - `ContentPresenter_ContentPresenter_Content_DataContext_Uno_UI_Samples_Content_UITests_ContentPresenter_ContentPresenter_Content_DataContext`
  - `ContentPresenter_ContentPresenter_ImplicitContent_UITests_Shared_Windows_UI_Xaml_Controls_ContentPresenter_ContentPresenter_ImplicitContent`
  - `Default_UITests_Windows_UI_Xaml_Controls_ComboBox_ComboBox_Dropdown_Background_4418_UITests_Windows_UI_Xaml_Controls_ComboBox_ComboBox_Dropdown_Background_4418`
  - `Default_UITests_Windows_UI_Xaml_Controls_CommandBar_CommandBar_Native_Frame_UITests_Windows_UI_Xaml_Controls_CommandBar_CommandBar_Native_Frame`
  - `Flyouts_Flyout_Attached_UITests_Shared_Windows_UI_Xaml_Controls_Flyout_Flyout_Attached`
  - `Flyouts_Flyout_ButtonInContent_UITests_Shared_Windows_UI_Xaml_Controls_Flyout_Flyout_ButtonInContent`
  - `Focus_Uno_UI_Samples_Content_UITests_FocusTests_FocusManager_FocusDirection_Uno_UI_Samples_Content_UITests_FocusTests_FocusManager_FocusDirection`
  - `Fonts_UITests_Windows_UI_Xaml_Media_FontTests_DynamicFont_UITests_Windows_UI_Xaml_Media_FontTests_DynamicFont`
  </details>

* `skia-linux-screenshots`: **78** changed over 2250

  <details>
  <summary>🚨🚨 Comparison Details (first 20) 🚨🚨</summary>

  - `Buttons.png-dark`
  - `DisplayInformation.png-dark`
  - `Buttons.png`
  - `DropDownButtonPage.png-dark`
  - `DropDownButtonPage.png`
  - `Examples.png`
  - `ExpanderColorValidationPage.png-dark`
  - `ExpanderColorValidationPage.png`
  - `ClipboardTests.png`
  - `ImageIconPage.png-dark`
  - `ImageIconPage.png`
  - `Gamepad_CurrentReading.png-dark`
  - `Gamepad_CurrentReading.png`
  - `Gamepad_Enumeration.png-dark`
  - `Gamepad_Enumeration.png`
  - `ImageBrushInList.png-dark`
  - `ImageBrushInList.png`
  - `Image_Stretch_None.png-dark`
  - `Image_Stretch_None.png`
  - `DoubleImageBrushInList.png-dark`
  </details>

* `skia-windows-screenshots`: **251** changed over 2250

  <details>
  <summary>🚨🚨 Comparison Details (first 20) 🚨🚨</summary>

  - `DoubleAnimation opacity final state.png-dark`
  - `DoubleAnimation opacity final state.png`
  - `Brushes_ImplicitConvert.png`
  - `ButtonClippingTestsControl.png-dark`
  - `ImagesInlineInFlipView.png-dark`
  - `ImagesInlineInFlipView.png`
  - `BorderImageBrushRelativeTransform.png-dark`
  - `DynamicBrushes_On_Shapes.png-dark`
  - `DynamicBrushes_On_Shapes.png`
  - `DynamicFont.png-dark`
  - `DynamicFont.png`
  - `ClosedFigure.png`
  - `ColorAnimationUsingKeyFrames_Fill.png-dark`
  - `CompositionNineGridBrush_Source_Changes.png-dark`
  - `CompositionNineGridBrush_Source_Changes.png`
  - `ImageUniformWithinScrollViewer.png-dark`
  - `ImageUniformWithinScrollViewer.png`
  - `Image_ScrollView_ChangeView.png-dark`
  - `Image_ScrollView_ChangeView.png`
  - `Brushes_ImplicitConvert.png-dark`
  </details>

* `wasm`: **46** changed over 1053

  <details>
  <summary>🚨🚨 Comparison Details (first 20) 🚨🚨</summary>

  - `UITests.Shared.Windows_UI.Xaml_Automation.AutomationProperties_AutomationId`
  - `UITests.Shared.Windows_UI_Xaml_Controls.MediaPlayerElement.MediaPlayerElement_Ogg_Extension`
  - `UITests.Windows_UI_Xaml_Controls.ComboBox.ComboBox_Disabled`
  - `UITests.Windows_UI_Xaml_Controls.BorderTests.BorderWithNullBrushAndNonZeroThickness`
  - `UITests.Windows_UI_Xaml_Controls.ListView.ListViewItem_IsEnabled`
  - `UITests.Windows_UI_Xaml_Controls.ListView.ListView_DisplayMemberPath`
  - `Uno.UI.Samples.Content.UITests.GridTestsControl.Grid_InsideStackPanel_InsideButton`
  - `GenericApp.Views.Samples.Shared.Content.UITests.GridViewMultipleSelectionMode`
  - `SamplesApp.Wasm.Windows_UI_Xaml_Controls.ComboBox.ComboBox_CornerRadius`
  - `SamplesApp.Wasm.Windows_UI_Xaml_Controls.ComboBox.ComboBox_Corners`
  - `UITests.Shared.Windows_UI_Xaml_Controls.ContentDialogTests.ContentDialog_Closing`
  - `UITests.Windows_UI_Xaml_Controls.ImageTests.BitmapImage_vs_SvgImageSource`
  - `SamplesApp.Windows_UI_Xaml_Controls.ListView.ListView_ItemContainerStyleSelector`
  - `UITests.Windows_UI_Xaml_Controls.BorderTests.PanelWithNullBrushAndNonZeroThickness`
  - `SamplesApp.Windows_UI_Xaml_Controls.ListView.ListViewLegacy`
  - `UITests.Shared.Microsoft_UI_Xaml_Controls.ExpanderTests.WinUIExpanderPage`
  - `UITests.Windows_UI_Xaml_Controls.Repeater.ItemsRepeater_Nested`
  - `UITests.Windows_UI_Xaml_Input.Keyboard.Keyboard_Events`
  - `Uno.UI.Samples.Content.UITests.ButtonTestsControl.Button_Enabled_Control_Disabled`
  - `Uno.UI.Samples.Content.UITests.ButtonTestsControl.CheckBox_Button`
  </details>

* `wasm-automated-net9.0-WinUI-Benchmarks-automated`: **1** changed over 1

  <details>
  <summary>🚨🚨 Comparison Details (first 20) 🚨🚨</summary>

  - `RunBenchmarks_Benchmarks_Shared_Controls_BenchmarkDotNetTestsPage`
  </details>

* `wasm-automated-net9.0-WinUI-Default-automated`: **13** changed over 876

  <details>
  <summary>🚨🚨 Comparison Details (first 20) 🚨🚨</summary>

  - `Default_StrokeThickness_MyLine`
  - `Default_StrokeThickness_MyPolyline`
  - `Default_StrokeThickness_MyEllipse`
  - `Default_StrokeThickness_MyPolygon`
  - `Default_StrokeThickness_MyRect`
  - `TestProgressRing_InitialState_UITests_Microsoft_UI_Xaml_Controls_ProgressRing_WinUIProgressRing_Features`
  - `When_StretchAndAlignmentNone_ImageBrush-50-50-None-XLeft-YBottom`
  - `When_SingleSelectionWithoutItemClick_Then_PointersEvents_UITests_Windows_UI_Xaml_Controls_ListView_ListView_Selection_Pointers`
  - `ListView_SelectedItems_SamplesApp_Windows_UI_Xaml_Controls_ListView_ListViewSelectedItems`
  - `Default_StrokeThickness_MyPath`
  - `SequentialAnimations_SamplesApp_Windows_UI_Xaml_Media_Animation_SequentialAnimationsPage`
  - `ToggleCollapsedStateToVerifyClipping_UITests_Shared_Microsoft_UI_Xaml_Controls_ExpanderTests_ExpanderColorValidationPage`
  - `When_ExtendedSelectionWithoutItemClick_Then_PointersEvents_UITests_Windows_UI_Xaml_Controls_ListView_ListView_Selection_Pointers`
  </details>

* `wasm-automated-net9.0-WinUI-RuntimeTests-0`: **0** changed over 1

* `wasm-automated-net9.0-WinUI-RuntimeTests-1`: **1** changed over 1

  <details>
  <summary>🚨🚨 Comparison Details (first 20) 🚨🚨</summary>

  - `RunRuntimeTests_SamplesApp_Samples_UnitTests_UnitTestsPage`
  </details>

* `wasm-automated-net9.0-WinUI-RuntimeTests-2`: **1** changed over 1

  <details>
  <summary>🚨🚨 Comparison Details (first 20) 🚨🚨</summary>

  - `RunRuntimeTests_SamplesApp_Samples_UnitTests_UnitTestsPage`
  </details>

</details>

MartinZikmund commented:
I am a bit worried about setting `UIViewControllerBasedStatusBarAppearance` to `false` (and didn't know this was already set to `false` in template) - would this mean we would not align with dark/light theme changes (e.g. the foreground could be out of sync unless the user explicitly fixes that?)

kazo0 commented on src/SamplesApp/SamplesApp.Skia.netcoremobile/iOS/Info.plist:
```diff
@@ -68,6 +68,8 @@
 	</array>
 	<key>UIApplicationSupportsIndirectInputEvents</key>
 	<true/>
+	<key>UIViewControllerBasedStatusBarAppearance</key>
+	<false/>
```
We should add some docs surrounding the StatusBar API that mention that this is required in the Info.plist

Similar to this: https://platform.uno/docs/articles/external/uno.toolkit.ui/doc/helpers/StatusBar-extensions.html#remarks

rajamatt replied:
There seems to be a lot of (outdated?) docs surrounding the StatusBar:
https://platform.uno/docs/articles/external/figma-docs/components/status-bar.html - StatusBar with OS property?
https://platform.uno/docs/articles/guides/status-bar-theme-color.html - is all this code still needed when switching themes?

And then there's the ones below using Toolkit StatusBarExtensions which we'll have to clean up or mention that starting with Uno 6.0.xxx this is available in uno core?
https://platform.uno/docs/articles/external/uno.toolkit.ui/doc/helpers/StatusBar-extensions.html
https://platform.uno/docs/articles/external/uno.chefs/doc/toolkit/StatusBarExtension.html

Maybe we should decide what to do with these docs and then create a separate issue "clean up statusbar docs"

kazo0 commented on src/Uno.UWP/UI/ViewManagement/StatusBar/StatusBar.Android.cs:
```diff
@@ -13,6 +14,10 @@ namespace Windows.UI.ViewManagement
 {
 	public sealed partial class StatusBar
 	{
+		public bool IsForegroundColorSet { get; set; }
+		public bool IsSettingBackgroundColor { get; set; }
+		public bool IsInitializingVisibility { get; set; }
```
These shouldn't be public as they are not part of the [WinUI StatusBar APIs](https://learn.microsoft.com/en-us/uwp/api/windows.ui.viewmanagement.statusbar?view=winrt-26100)

kazo0 replied:
Also I'm not sure about using these properties. Couldn't we check if ForegroundColor and/or BackgroundColor is `null`?

rajamatt replied:
Got it. Yeah there might be a way to use ForegroundColor/BackgroundColor but I wasn't able to find a good way. I'll see if I can remove these public properties.

rajamatt replied:
I ended up finding convincing ways to get around needing these public properties.

For `IsSettingForegroundColor`, it was just because I didn't realize that `GetStatusBarForegroundType()` never actually returned null. Added nullability there.

For `IsInitializingVisibility`, we don't actually need to call UpdateUIVisibility() for the BackgroundColor. So we don't need this one at all anymore.

For BackgroundColor, since we don't need `IsInitializingVisibility` we can just check if the BackgroundColor is not null and everything works.

kazo0 commented on src/Uno.UWP/UI/ViewManagement/StatusBar/StatusBar.Android.cs:
```diff
@@ -95,6 +100,7 @@ private IAsyncAction SetVisibilityAsyncCore(bool visible)
 			{
 				CoreDispatcher.CheckThreadAccess();
 				_isShown = visible;
+				IsInitializingVisibility = true;
```
I'm not sure what this property is needed for. How come we need it before we clear the inset.top?

rajamatt replied:
tldr: Extra padding flickered on first BackgroundColor set because `SetVisibilityAsyncCore` ran before `IsSettingBackgroundColor` took effect.

This one was a bit hard to dig out. The background extra padding we talked about was fixed only with `IsSettingBackgroundColor`, EXCEPT every time we set the BackgroundColor there is this `SetVisibilityAsyncCore` that is called before even getting to the BackgroundColor part. So the first time setting the BackgroundColor there was the extra padding visible for a few seconds, the content would jump when it was removed when finally hitting the `IsSettingBackgroundColor` assignment.

rajamatt committed:
fix: android padding and iOS landscape

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-20687/docs/index.html

unodevops commented:
🤖 Your WebAssembly Skia Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-20687/wasm-skia-net9/index.html

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-20687/docs/index.html

rajamatt commented on src/Uno.UWP/UI/ViewManagement/StatusBar/StatusBar.Android.cs:
```diff
@@ -110,6 +118,38 @@ private IAsyncAction SetVisibilityAsyncCore(bool visible)
+				return;
+			}
+
+			if ((int)Build.VERSION.SdkInt >= 35)
```
The CI was failing because it didn't recognize the BuildVersionCodes.VanillaIceCream. I just looked for that CI build so I can link it here but I can't find it on Azure.

I wasn't sure if we needed to update packages used by CI or not. I found this in the uno code so that's why I changed it to 35 https://github.com/unoplatform/uno/blob/46d302a0e64735214be9669e78ab299a6d218c6b/src/Uno.UI/FeatureConfiguration.cs#L1015

unodevops commented:
🤖 Your WebAssembly Skia Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-20687/wasm-skia-net9/index.html

rajamatt commented on src/Uno.UWP/UI/ViewManagement/StatusBar/StatusBar.Android.cs:
```diff
@@ -110,6 +118,38 @@ private IAsyncAction SetVisibilityAsyncCore(bool visible)
+
+			if ((int)Build.VERSION.SdkInt >= 35)
+			{
+				ViewCompat.SetOnApplyWindowInsetsListener(decorView, new InsetsListener(this));
```
I added a WIP version to this PR but the extra padding is back so I'll have to take another look at it

rajamatt replied:
Fixed now, we were calling UpdateUIVisibility() when we don't need to

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-20687/docs/index.html

unodevops commented:
🤖 Your WebAssembly Skia Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-20687/wasm-skia-net9/index.html

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-20687/docs/index.html

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-20687/docs/index.html

unodevops commented:
🤖 Your WebAssembly Skia Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-20687/wasm-skia-net9/index.html

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-20687/docs/index.html

rajamatt commented:
> @rajamatt I'm still seeing the issue for iOS on orientation change:
> 
>  Simulator.Screen.Recording.-.iPhone.16.Pro.Max.-.2025-06-09.at.16.36.05.mp4

Fixed!

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-20687/docs/index.html

unodevops commented:
🤖 Your WebAssembly Skia Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-20687/wasm-skia-net9/index.html

rajamatt committed:
fix: only android 15 needs insets to be reset

unodevops commented:
🤖 Your WebAssembly Skia Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-20687/wasm-skia-net9/index.html

rajamatt committed:
fix: android null backgroundcolor

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-20687/docs/index.html

unodevops commented:
🤖 Your WebAssembly Skia Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-20687/wasm-skia-net9/index.html

kazo0 commented:
@rajamatt the Android test failures are making me think we are unexpectedly causing a shift in the layout of the window content and now the coordinates for the test are wrong when checking for specific pixel locations and colors

kazo0 commented:
/azp run

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-20687/docs/index.html

rajamatt committed:
docs: add to viewmanagement and remove old statusbar docs

unodevops commented:
🤖 Your WebAssembly Skia Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-20687/wasm-skia-net9/index.html

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-20687/docs/index.html

rajamatt commented on src/SamplesApp/UITests.Shared/Windows_UI_ViewManagement/StatusBarColorTests.xaml.cs:
```diff
@@ -0,0 +1,88 @@
+
+namespace UITests.Shared.Windows_UI_ViewManagement
+{
+	[SampleControlInfo("Windows.UI.ViewManagement", "StatusBar_Color", description: "Allows changing status bar color dynamically")]
```
Made description clearer on what to test.

rajamatt commented:
@kazo0 for docs I added a StatusBar mention on the uno Windows ViewManagement page

And removed the other existing page about changing statusbar theming on mobile since it was heavily outdated

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-20687/docs/index.html

Kunal22shah committed:
chore: handle reset bg color

Kunal22shah committed:
chore: fix condt. check

rajamatt committed:
chore: refactor disposing android

unodevops commented:
🤖 Your WebAssembly Skia Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-20687/wasm-skia-net9/index.html

unodevops commented:
⚠️⚠️ The build [168781](https://dev.azure.com/uno-platform//Uno%20Platform/_build/results?buildId=168781) has failed on Uno.UI - CI.

rajamatt committed:
fix: setting null backgroundcolor first

kazo0 committed:
chore: fix reset statusbar

unodevops commented:
🤖 Your WebAssembly Skia Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-20687/wasm-skia-net9/index.html

kazo0 committed:
chore: restore UpdateSystemUiVisibility call

unodevops commented:
🤖 Your WebAssembly Skia Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-20687/wasm-skia-net9/index.html

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-20687/docs/index.html

kazo0 committed:
chore: Sample fixes

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-20687/docs/index.html

rajamatt committed:
chore: add missing endif

unodevops commented:
🤖 Your WebAssembly Skia Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-20687/wasm-skia-net9/index.html

rajamatt committed:
chore: missing semicolon

unodevops commented:
🤖 Your WebAssembly Skia Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-20687/wasm-skia-net9/index.html

unodevops commented:
⚠️⚠️ The build [168922](https://dev.azure.com/uno-platform//Uno%20Platform/_build/results?buildId=168922) has failed on Uno.UI - CI.

unodevops commented:
🤖 Your Docs stage site is ready! Visit it here: https://unodocsprstaging.z13.web.core.windows.net/pr-20687/docs/index.html

unodevops commented:
🤖 Your WebAssembly Skia Sample App stage site is ready! Visit it here: https://unowasmprstaging.z20.web.core.windows.net/pr-20687/wasm-skia-net9/index.html

nventive-devops commented:
The [build 168987](https://dev.azure.com/uno-platform/Uno%20Platform/_build/results?buildId=168987) found UI Test snapshots differences: `android-28-net8`: **32**, `android-28-net8-Snap`: **51**, `ios`: **222**, `ios-Snap`: **988**, `skia-linux-screenshots`: **75**, `skia-windows-screenshots`: **295**, `wasm`: **50**, `wasm-automated-net9.0-WinUI-Benchmarks-automated`: **0**, `wasm-automated-net9.0-WinUI-Default-automated`: **10**, `wasm-automated-net9.0-WinUI-RuntimeTests-0`: **0**, `wasm-automated-net9.0-WinUI-RuntimeTests-1`: **1**, `wasm-automated-net9.0-WinUI-RuntimeTests-2`: **0**

<details>
<summary>Details</summary>


* `android-28-net8`: **32** changed over 824

  <details>
  <summary>🚨🚨 Comparison Details (first 20) 🚨🚨</summary>

  - `Detereminate_ProgressRing_Validation25_[#FF0000_#008000_#FF0000_#FF0000]_Progress-Ring-Value-25`
  - `ListView_ListViewWithHeader_InitializesTest_SamplesApp_Windows_UI_Xaml_Controls_ListView_HorizontalListViewGrouped`
  - `NativeCommandBar_Size_Uno_UI_Samples_Content_UITests_CommandBar_CommandBar_Dynamic`
  - `SelectionOnLoad_UITests_Microsoft_UI_Xaml_Controls_RadioButtonsTests_RadioButtonsInitialLoadSelected`
  - `SelectionTest_UITests_Microsoft_UI_Xaml_Controls_RadioButtonsTests_RadioButtonsPage`
  - `WebView_NavigateToAnchor_Initial`
  - `UpDownEnabledTest_UITests_Shared_Microsoft_UI_Xaml_Controls_NumberBoxTests_NumberBoxPage`
  - `When_NoSelectionWithItemClick_Then_PointersEvents_UITests_Windows_UI_Xaml_Controls_ListView_ListView_Selection_Pointers`
  - `When_NoSelection_Then_PointersEvents_UITests_Windows_UI_Xaml_Controls_ListView_ListView_Selection_Pointers`
  - `When_SingleSelectionWithItemClick_Then_PointersEvents_UITests_Windows_UI_Xaml_Controls_ListView_ListView_Selection_Pointers`
  - `When_SingleSelectionWithoutItemClick_Then_PointersEvents_UITests_Windows_UI_Xaml_Controls_ListView_ListView_Selection_Pointers`
  - `DecimalFormatterTest_UITests_Shared_Microsoft_UI_Xaml_Controls_NumberBoxTests_NumberBoxPage`
  - `DismissablaPopups_ForwardOrder_Validation_UITests_Shared_Windows_UI_Xaml_Controls_Popup_Popup_LightDismiss`
  - `ProgressRing_IsEnabled_Running_UITests_Windows_UI_Xaml_Controls_ProgressRing_WindowsProgressRing_GH1220`
  - `ProgressRing_Visibility_Collapsed_UITests_Windows_UI_Xaml_Controls_ProgressRing_WindowsProgressRing_GH1220`
  - `SequentialAnimations_SamplesApp_Windows_UI_Xaml_Media_Animation_SequentialAnimationsPage`
  - `When_Clip_Is_Set_On_Child_Element_validation`
  - `When_Clip_Is_Set_On_Container_Element_UITests_Windows_UI_Xaml_Clipping_Clipping652`
  - `When_Parent_PointerMoved_After_drag_on_non-scrolling_ScrollViewer`
  - `When_Parent_PointerMoved_After_drag_on_ScrollViewer_-_touch`
  </details>

* `android-28-net8-Snap`: **51** changed over 1071

  <details>
  <summary>🚨🚨 Comparison Details (first 20) 🚨🚨</summary>

  - `ListView_ListViewSelectedItems_SamplesApp_Windows_UI_Xaml_Controls_ListView_ListViewSelectedItems`
  - `NavigationView_UITests_Microsoft_UI_Xaml_Controls_NavigationViewTests_NavigationViewBasicPage_UITests_Microsoft_UI_Xaml_Controls_NavigationViewTests_NavigationViewBasicPage`
  - `MediaPlayerElement_Using_3gp_UITests_Shared_Windows_UI_Xaml_Controls_MediaPlayerElement_MediaPlayerElement_3gp_Extension`
  - `MediaPlayerElement_Using_mp3_Audio_only_UITests_Shared_Windows_UI_Xaml_Controls_MediaPlayerElement_MediaPlayerElement_Mp3_Extension`
  - `MediaPlayerElement_Using_ogg_UITests_Shared_Windows_UI_Xaml_Controls_MediaPlayerElement_MediaPlayerElement_Ogg_Extension`
  - `ListView_ListView_TransformsOnList_UITests_Shared_Windows_UI_Xaml_Controls_ListView_ListView_TransformsOnList`
  - `ListView_ListView_With_ListViews_Count_Measure_UITests_Shared_Windows_UI_Xaml_Controls_ListView_ListView_With_ListViews_Count_Measure`
  - `Icons_SamplesApp_Windows_UI_Xaml_Shapes_PathTestsControl_PathIcon_SamplesApp_Windows_UI_Xaml_Shapes_PathTestsControl_PathIcon`
  - `ItemsRepeater_UITests_Windows_UI_Xaml_Controls_Repeater_StackLayout_Simple_UITests_Windows_UI_Xaml_Controls_Repeater_StackLayout_Simple`
  - `MediaPlayerElement_Mini_player_UITests_Shared_Windows_UI_Xaml_Controls_MediaPlayerElement_MediaPlayerElement_Minimal`
  - `Icons_UITests_Microsoft_UI_Xaml_Controls_ImageIconTests_ImageIconPage_UITests_Microsoft_UI_Xaml_Controls_ImageIconTests_ImageIconPage`
  - `Icons_UITests_Shared_Windows_UI_Xaml_Controls_BitmapIconTests_BitmapIcon_Foreground_UITests_Shared_Windows_UI_Xaml_Controls_BitmapIconTests_BitmapIcon_Foreground`
  - `ListView_ListView_Explicit_Items_UITests_Shared_Windows_UI_Xaml_Controls_ListView_ListView_Explicit_Items`
  - `ListView_ListView_FirstLastCacheIndex_SamplesApp_Windows_UI_Xaml_Controls_ListView_ListView_FirstLastCacheIndex`
  - `ListView_ListView_Header_DataContextChanging_UITests_Shared_Windows_UI_Xaml_Controls_ListView_Header_DataContextChanging`
  - `ListView_ListView_Image_SamplesApp_Windows_UI_Xaml_Controls_ListView_ListView_Image`
  - `Image_Uno_UI_Samples_UITests_Image_Image_Stretch_Alignment_Bigger_Uno_UI_Samples_UITests_Image_Image_Stretch_Alignment_Bigger`
  - `NavigationView_MUXControlsTestApp_NavigationViewMenuItemStretchPage_MUXControlsTestApp_NavigationViewMenuItemStretchPage`
  - `MUX_UITests_Shared_Microsoft_UI_Xaml_Controls_TreeViewTests_TreeViewPage_UITests_Shared_Microsoft_UI_Xaml_Controls_TreeViewTests_TreeViewPage`
  - `TeachingTip_UITests_Microsoft_UI_Xaml_Controls_TeachingTipTests_TeachingTipPage_UITests_Microsoft_UI_Xaml_Controls_TeachingTipTests_TeachingTipPage`
  </details>

* `ios`: **222** changed over 255

  <details>
  <summary>🚨🚨 Comparison Details (first 20) 🚨🚨</summary>

  - `CheckBox_IsEnabled_Validation_Uno_UI_Samples_Content_UITests_ButtonTestsControl_CheckBox_IsEnabled_Automated`
  - `Change_Manipulation_Property_UITests_Shared_Windows_UI_Xaml_Controls_BorderTests_Border_Clipped_Change_Property`
  - `ComboBoxTests_ToggleDisabled_ComboBox_Disabled`
  - `ComboBoxTests_Fullscreen_Popup_iOS_Closed`
  - `DatePicker_Flyout_DatePicker_-_Flyout`
  - `Image_Source_Nullify_image_source_nullify_empty`
  - `Image_Margins_Identical_WriteableBitmap_Invalidate_-_Result`
  - `Keyboard_Textbox_NoScrollViewer_Validation_Uno_UI_Samples_Content_UITests_TextBoxControl_Input_Test_NoScrollViewer_Automated`
  - `Keyboard_Textbox_NoScrollViewer_Validation_1_-_Remove_Focus_on_filledTextBox`
  - `ListView_ExpandableItemLarge_ExpandHeader_Validation_SamplesApp_Windows_UI_Xaml_Controls_ListView_ListView_Expandable_Item_Large`
  - `ListView_ExpandableItemLarge_ExpandHeaderWithSingleItem_Validation_Expanded_State`
  - `ListView_ObservableCollection_Creation_Count_Scrolled2`
  - `ListView_ExpandableItem_ExpandSingleItem_SamplesApp_Windows_UI_Xaml_Controls_ListView_ListView_Expandable_Item`
  - `ListView_Weird_Measure_During_Arrange_UITests_Shared_Windows_UI_Xaml_Controls_ListView_ListView_Weird_Measure`
  - `Screenshots_Image_Stretch_Alignment_Equal_Mode-2`
  - `Screenshots_Image_Stretch_Algmnt_Inf_Vertical_Mode-0`
  - `Screenshots_Image_Stretch_Alignment_Equal_Mode-3`
  - `Screenshots_Image_Stretch_Algmnt_Inf_Vertical_Mode-1`
  - `ScrollViewer_Margin_UITests_Windows_UI_Xaml_Controls_ScrollViewerTests_ScrollViewer_Margin`
  - `ScrollViewer_Content_Margin_UITests_Windows_UI_Xaml_Controls_ScrollViewerTests_ScrollViewer_Content_Margin`
  </details>

* `ios-Snap`: **988** changed over 988

  <details>
  <summary>🚨🚨 Comparison Details (first 20) 🚨🚨</summary>

  - `Border_AutoBorderStretchwithleftmargin_Uno_UI_Samples_UITests_BorderTestsControl_AutoBorderStretchwithleftmargin`
  - `Border_Border_Simple_Uno_UI_Samples_UITests_BorderTestsControl_Border_Simple`
  - `Border_UITests_Windows_UI_Xaml_Controls_BorderTests_PanelWithNullBrushAndNonZeroThickness_UITests_Windows_UI_Xaml_Controls_BorderTests_PanelWithNullBrushAndNonZeroThickness`
  - `Brushes_ImageBrushShapeStretchesAlignments_UITests_Windows_UI_Xaml_Media_ImageBrushTests_ImageBrushShapeStretchesAlignments`
  - `Brushes_RectangleStretchFill_Uno_UI_Samples_UITests_ImageBrushTestControl_RectangleStretchFill`
  - `Brushes_UITests_Windows_UI_Xaml_Media_BrushesTests_RevealBrush_Fallback_UITests_Windows_UI_Xaml_Media_BrushesTests_RevealBrush_Fallback`
  - `Brushes_Uno_UI_Samples_UITests_ImageBrushTestControl_ImageBrushAlignmentXY2_Uno_UI_Samples_UITests_ImageBrushTestControl_ImageBrushAlignmentXY2`
  - `Buttons_Hyperlink_Disabled_nVentive_Umbrella_Views_UI_Samples_Content_UITests_ButtonTestsControl_Hyperlink_Disabled`
  - `Buttons_UITests_Microsoft_UI_Xaml_Controls_RadioButtonsTests_RadioButtonsInitialLoadSelected_UITests_Microsoft_UI_Xaml_Controls_RadioButtonsTests_RadioButtonsInitialLoadSelected`
  - `Buttons_UITests_Microsoft_UI_Xaml_Controls_RadioButtonsTests_RadioButtonsPage_UITests_Microsoft_UI_Xaml_Controls_RadioButtonsTests_RadioButtonsPage`
  - `ComboBox_SamplesApp_Wasm_Windows_UI_Xaml_Controls_ComboBox_ComboBox_FullScreen_Popup_SamplesApp_Wasm_Windows_UI_Xaml_Controls_ComboBox_ComboBox_FullScreen_Popup`
  - `ComboBox_SamplesApp_Wasm_Windows_UI_Xaml_Controls_ComboBox_ComboBox_Header_SamplesApp_Wasm_Windows_UI_Xaml_Controls_ComboBox_ComboBox_Header`
  - `ContentControl_ContentControl_FindName_Uno_UI_Samples_Content_UITests_ContentControlTestsControl_ContentControl_FindName`
  - `ContentControl_ContentControl_Inheritance_Uno_UI_Samples_Content_UITests_ContentControlTestsControl_ContentControl_Inheritance`
  - `Default_UITests_Shared_Windows_UI_Xaml_Controls_ListView_ListView_Weird_Measure_UITests_Shared_Windows_UI_Xaml_Controls_ListView_ListView_Weird_Measure`
  - `Default_UITests_Shared_Windows_UI_Xaml_Controls_Popup_Popup_Overlay_On_UITests_Shared_Windows_UI_Xaml_Controls_Popup_Popup_Overlay_On`
  - `Dialogs_UITests_Shared_MessageDialogTests_MessageDialogTest_UITests_Shared_MessageDialogTests_MessageDialogTest`
  - `Dialogs_UITests_Windows_UI_Xaml_Controls_ContentDialogTests_ContentDialog_Dialogs_UITests_Windows_UI_Xaml_Controls_ContentDialogTests_ContentDialog_Dialogs`
  - `Flyouts_Namescope_UITests_Shared_Windows_UI_Xaml_Controls_FlyoutTests_Flyout_Namescope`
  - `Flyouts_UITests_Shared_Microsoft_UI_Xaml_Controls_RadioMenuFlyoutItemTests_RadioMenuFlyoutItemPage_UITests_Shared_Microsoft_UI_Xaml_Controls_RadioMenuFlyoutItemTests_RadioMenuFlyoutItemPage`
  </details>

* `skia-linux-screenshots`: **75** changed over 2268

  <details>
  <summary>🚨🚨 Comparison Details (first 20) 🚨🚨</summary>

  - `Examples.png-dark`
  - `Examples.png`
  - `ExpanderColorValidationPage.png-dark`
  - `ExpanderColorValidationPage.png`
  - `DropDownButtonPage.png-dark`
  - `DropDownButtonPage.png`
  - `Focus_FocusVisual_Properties.png-dark`
  - `Focus_FocusVisual_Properties.png`
  - `ButtonClippingTestsControl.png-dark`
  - `CalendarView_Theming.png-dark`
  - `ClipboardTests.png-dark`
  - `ClipboardTests.png`
  - `DisplayInformation.png-dark`
  - `DisplayInformation.png`
  - `ImageBrushInList.png-dark`
  - `ImageBrushInList.png`
  - `ImageBrushLocalAsset.png-dark`
  - `ImageBrushLocalAsset.png`
  - `Buttons.png-dark`
  - `Buttons.png`
  </details>

* `skia-windows-screenshots`: **295** changed over 2268

  <details>
  <summary>🚨🚨 Comparison Details (first 20) 🚨🚨</summary>

  - `AutoSuggestBox_Reason.png-dark`
  - `AutoSuggestBox_Reason.png`
  - `ComboBox_Dropdown_Background_4418.png-dark`
  - `ComboBox_Dropdown_Background_4418.png`
  - `DoubleImageBrushInList.png-dark`
  - `DoubleImageBrushInList.png`
  - `GetFocus.png-dark`
  - `GetFocus.png`
  - `AutoSuggestBox_BitmapIcon.png-dark`
  - `AutoSuggestBox_BitmapIcon.png`
  - `BitmapImage_vs_SvgImageSource.png`
  - `Border_BorderThickness.png-dark`
  - `Border_BorderThickness.png`
  - `Border_CornerRadius_Clipping2.png`
  - `Border_CornerRadius_Gradient.png-dark`
  - `CalendarDatePicker_Basics.png-dark`
  - `Focus_FocusVisual.png-dark`
  - `Focus_FocusVisual.png`
  - `Focus_FocusVisual_Properties.png-dark`
  - `Focus_FocusVisual_Properties.png`
  </details>

* `wasm`: **50** changed over 1055

  <details>
  <summary>🚨🚨 Comparison Details (first 20) 🚨🚨</summary>

  - `GenericApp.Views.Samples.Shared.Content.UITests.GridViewMultipleSelectionMode`
  - `initial_state`
  - `SamplesApp.Windows_UI_Xaml_Controls.ListView.ListViewLegacy`
  - `UITests.Microsoft_UI_Xaml_Controls.WebView2Tests.WebView2_NavigationProperties`
  - `UITests.Windows_UI_Xaml_Controls.ListView.ListViewItem_IsEnabled`
  - `UITests.Shared.Windows_UI_Xaml_Controls.Button.Button_Events`
  - `UITests.Shared.Windows_UI_Xaml_Controls.Button.Overlapped_Buttons`
  - `UITests.Shared.Windows_UI_Xaml_Media_Animation.SetTargetProperty`
  - `UITests.Windows_UI_Xaml.FocusTests.Focus_FocusState`
  - `UITests.Windows_UI_Xaml_Controls.ListView.ListViewItem_Click_Focus`
  - `Uno.UI.Samples.Content.UITests.ButtonTestsControl.AppBar_KeyBoard`
  - `Uno.UI.Samples.Content.UITests.ButtonTestsControl.Button_Enabled_Control_Disabled`
  - `Uno.UI.Samples.Content.UITests.CommandBar.CommandBar_With_Long_Sentences`
  - `Uno.UI.Samples.Content.UITests.GridTestsControl.Grid_InsideStackPanel_InsideButton`
  - `Uno.UI.Samples.Content.UITests.ContentControlTestsControl.ContentControl_Inheritance`
  - `SamplesApp.Wasm.Windows_UI_Xaml_Controls.ListView.ListView_IsSelected`
  - `Uno.UI.Samples.Content.UITests.GridTestsControl.Grid_with_UserControlMargin`
  - `SamplesApp.Wasm.Windows_UI_Xaml_Controls.ComboBox.ComboBox_CornerRadius`
  - `SamplesApp.Windows_UI_Xaml_Controls.ListView.ListView_Inside_ListView`
  - `SamplesApp.Windows_UI_Xaml_Controls.ListView.UndefinedHeightListView`
  </details>

* `wasm-automated-net9.0-WinUI-Benchmarks-automated`: **0** changed over 1

* `wasm-automated-net9.0-WinUI-Default-automated`: **10** changed over 876

  <details>
  <summary>🚨🚨 Comparison Details (first 20) 🚨🚨</summary>

  - `ListView_SelectedItems_SamplesApp_Windows_UI_Xaml_Controls_ListView_ListViewSelectedItems`
  - `When_MultipleSelectionWithoutItemClick_Then_PointersEvents_UITests_Windows_UI_Xaml_Controls_ListView_ListView_Selection_Pointers`
  - `Default_StrokeThickness_MyPolyline`
  - `Default_StrokeThickness_MyPath`
  - `Default_StrokeThickness_MyEllipse`
  - `Default_StrokeThickness_MyRect`
  - `TestProgressRing_InitialState_UITests_Microsoft_UI_Xaml_Controls_ProgressRing_WinUIProgressRing_Features`
  - `Default_StrokeThickness_MyLine`
  - `SequentialAnimations_SamplesApp_Windows_UI_Xaml_Media_Animation_SequentialAnimationsPage`
  - `Default_StrokeThickness_MyPolygon`
  </details>

* `wasm-automated-net9.0-WinUI-RuntimeTests-0`: **0** changed over 1

* `wasm-automated-net9.0-WinUI-RuntimeTests-1`: **1** changed over 1

  <details>
  <summary>🚨🚨 Comparison Details (first 20) 🚨🚨</summary>

  - `RunRuntimeTests_SamplesApp_Samples_UnitTests_UnitTestsPage`
  </details>

* `wasm-automated-net9.0-WinUI-RuntimeTests-2`: **0** changed over 1

</details>
